controller/bluegreen.go

```diff
@@ -248,6 +248,20 @@ func (c *Controller) scaleBlueGreen(rollout *v1alpha1.Rollout, newRS *appsv1.Rep
 		}
 	}
 
+	previewRS, _ := replicasetutil.GetReplicaSetByTemplateHash(allRS, rollout.Status.BlueGreen.PreviewSelector)
+	if previewRS != nil {
+		var previewReplicas int32
+		if rollout.Spec.Strategy.BlueGreenStrategy.PreviewReplicaCount != nil {
+			previewReplicas = *rollout.Spec.Strategy.BlueGreenStrategy.PreviewReplicaCount
+		} else {
+			previewReplicas = rolloutReplicas
```

Comment:
Is there a reason that you aren't setting the previewReplicas to rolloutReplicas as the default and simplifying the if else statement to just an if statement?

Reply:
Good suggestion.